config/example.sh

```diff
@@ -31,3 +33,5 @@ export DM_PRODUCTION_ADMIN_FRAMEWORK_MANAGER_USER_EMAIL=${DM_PRODUCTION_ADMIN_FR
 export DM_PRODUCTION_ADMIN_FRAMEWORK_MANAGER_USER_PASSWORD=${DM_PRODUCTION_ADMIN_FRAMEWORK_MANAGER_USER_PASSWORD:=Password1234}
 
 export DM_PAGINATION_LIMIT=${DM_PAGINATION_LIMIT:=5}
+
+export DM_DOCUMENTS_BUCKET_NAME=${DM_DOCUMENTS_BUCKET_NAME:=digitalmarketplace-documents-${DM_ENVIRONMENT}-${DM_ENVIRONMENT}}
```

Comment:
As discussed offline, this will need to go into the Jenkins job template.

Reply:
:+1: